CONTRIBUTING.md

```diff
@@ -86,7 +86,7 @@ Triage member or another Mappings Team member.
 
 **Final Comment Period**:
 - Snapshots: 3 days
-- Releases: 7 days
+- Releases: 5 days
```

Comment:
Hm, why is documentation  just about equal to refactor? I think 2/4 is good here instead of 3/5.